migration/FromMySqlToPostgreSql/MapDataTypes.php

```diff
@@ -164,7 +164,7 @@ private function __construct()
 
         'point' => [
             'increased_size'           => '',
```

Comment:
MySQL test case:


create table testGeoPoint (
        `ID` bigint(20) NOT NULL,
        `GEOPOINT` point DEFAULT NULL,
        `LATITUDE` decimal(19,10) DEFAULT NULL,
        `LONGITUDE` decimal(19,10) DEFAULT NULL
) ENGINE=InnoDB;
        

insert into testGeoPoint values ( 1, GeomFromText('POINT(-74.0445000000 40.6892000000 )' ), 40.6892000000, -74.0445000000 );